ViewModels/LeverPostingApplyViewModel.cs

```diff
@@ -59,6 +62,75 @@ public string ToJson
             }
         }
 
+        public MultipartFormDataContent ToDataFrom()
```

Comment:
SoundCloud isn't a fan of the length of this method (it's double what it recommends). Happy to chat through ideas on how we can reduce the length of it.